### ISSUE TITLE
arch package: incorrect sha256 checksum

### ISSUE THREAD

**Issue by draptik** [remotemobprogramming/mob#227]
Trying to update mobsh-bin to v2.2.0 fails due to incorrect sha256 checksum.

Error message:

```txt
❯ yay
...

:: 1 Packages to upgrade.
1  aur/mobsh-bin  2.1.0-1 -> 2.2.0-1
...

:: (1/1) Parsing SRCINFO: mobsh-bin
==> Making package: mobsh-bin 2.2.0-1 (Sun 09 Jan 2022 04:30:05 PM CET)
==> Retrieving sources...
  -> Found mob_v2.2.0_linux_amd64.tar.gz
==> Validating source_x86_64 files with sha256sums...
    mob_v2.2.0_linux_amd64.tar.gz ... FAILED
==> ERROR: One or more files did not pass the validity check!
 -> error downloading sources: mobsh-bin
         context: exit status 1


==> Making package: mobsh-bin 2.2.0-1 (Sun 09 Jan 2022 04:30:07 PM CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found mob_v2.2.0_linux_amd64.tar.gz
==> Validating source_x86_64 files with sha256sums...
    mob_v2.2.0_linux_amd64.tar.gz ... FAILED
==> ERROR: One or more files did not pass the validity check!
 -> error making: mobsh-bin
```

The provided checksum in `PKGBUILD` is:

```txt
sha256sums_x86_64=("8f9042e2210d8ac2c4d08443a11961b6246d99cedab86280ad85b79eb6632916")
```

The correct checksum is:

```txt
$ sha256sum mob_v2.2.0_linux_amd64.tar.gz
439101ab62429ba60e71a9471c0f39d9562f644b0d8460fdda5ff9316325abf9  mob_v2.2.0_linux_amd64.tar.gz
```

The `release.yaml` has this line:

https://github.com/remotemobprogramming/mob/blob/ca865a226205f5c112b9e3578063fa9c73945877/.github/workflows/release.yaml#L93

Maybe `needs.release.outputs.sha` gets the wrong environment variable?

**Comment by seanpoulter** [comment]
Uh oh. My apologies Patrick. I can have a look during the next 24-48 hours.

**Comment by seanpoulter** [comment]
I've got to step away, but my hypothesis is that I've used the checksum for MacOS. See https://github.com/remotemobprogramming/mob/blob/main/.github/workflows/release.yaml#L43.

**Comment by seanpoulter** [comment]
I can confirm that the checksum in `PKGBUILD` is from the MacOS (darwin) build:

![image](https://user-images.githubusercontent.com/2585460/148696443-15bf24b9-d0a4-439a-b79e-8c1e5a7fdf0d.png)

I've created PR #228 to fix the issue. Could you have a look @draptik?

**Comment by seanpoulter** [comment]
Should we fix the previous releases that may be affected since November?

**Comment by draptik** [comment]
@seanpoulter previous releases don't have the issue since the were build manually IIRC.
Also Arch Linux is a rolling release distro, so most people will use the most up-to-date version anyway.

**Comment by simonharrer** [comment]
released v2.2.1. 

@draptik @seanpoulter 
Can you please confirm that everything is okay with this release?

**Comment by draptik** [comment]
v2.2.1 can successfully be installed on Arch.
Thanks for the quick fix!
